dashboard/modules/job/cli.py

```diff
@@ -175,6 +175,14 @@ def job_cli_group():
     help="the quantity of GPUs to reserve for the entrypoint command, "
     "separately from any tasks or actors that are launched by it",
 )
+@click.option(
+    "--entrypoint-memory",
+    required=False,
+    type=float,
+    help="the amount of total available memory for workers requesting memory "
+    "for the entrypoint command, separately from any tasks or actors that are "
+    "launched by it",
```

Comment:
```suggestion
    help="the amount of memory to reserve "
    "for the entrypoint command, separately from any tasks or actors that are "
    "launched by it",
```
I think this is more accurate, since it doesn't apply to any Ray worker processes technically